django_temporalio/utils.py

```diff
@@ -0,0 +1,46 @@
+import fnmatch
+import os
+from importlib import import_module
+
+from django_temporalio.conf import settings, SettingIsNotSetError
+
+
+def autodiscover_modules(related_name_pattern: str):
+    """
+    Autodiscover modules matching the related name pattern in the base module.
+
+    Example for the following directory structure:
+
+    foo/
+        workflows.py
+        activities.py
+        bar/
+            bar_workflows.py
+            activities.py
+            baz/
+                workflows_baz.py
+                activities.py
+
+    Calling `autodiscover_modules('foo', '*workflows*')` will discover the following modules:
```

Comment:
This doc needs updating. It's not possible to pass two arguments to this function.